app/policies/rent_policy.rb

```diff
@@ -0,0 +1,30 @@
+class RentPolicy < ApplicationPolicy
+  class Scope
+    attr_reader :user, :scope
+
+    def initialize(user, scope)
+      @user  = user
+      @scope = scope
+    end
+
+    def resolve
+      scope.where(user_id: user.id)
+    end
+  end
+
+  def index?
+    user.present? && user == rent.user
```

Comment:
I don't think this is working.shouldn't you use `resource` instead of `rent` ?

Reply:
It is working fine. I've declared a private rent method below which returns the rent instance